### DISCUSSION TITLE
hi im getting this error can please tell how to deal with that thank you

### DISCUSSION THREAD

**Discussion by raju-b70** [learndevopsonline/learndevopsonline#1848] [Q&A]
![Screenshot (31)](https://github.com/learndevopsonline/learndevopsonline/assets/161826606/3ce0fd17-026a-4115-aa08-8746eadcaf18)

**Comment by chaithanya1812** [comment]
# Issue:- expense.conf file is missing
 I saw your repository there is no expense.conf file , 
Add expense.conf file to your code repository
